src/global.css

```diff
@@ -57,12 +63,36 @@ button{
   width: 100%;
 }
 
-.no-scrollbar::-webkit-scrollbar{
-    width: 0px;
-    background: transparent; /* Chrome/Safari/Webkit */
+.no-scrollbar::-webkit-scrollbar {
+  width: 0px;
+  background: transparent;
+  /* Chrome/Safari/Webkit */
 }
 
 .no-scrollbar {
-  scrollbar-width: none; /* Firefox */
-  -ms-overflow-style: none;  /* IE 10+ */
+  scrollbar-width: none;
+  /* Firefox */
+  -ms-overflow-style: none;
+  /* IE 10+ */
+}
+
+@media all and (min-width: 769px) {
+  body {
+    overflow: hidden;
+  }
+
+}
+
+.grid {
```

Comment:
This should probably be contained in Trade.svelte, not in global CSS.

Reply:
Done

Reply:
I don't see any new commits?

Reply:
> I don't see any new commits.

My apologies, I've pushed it now